src/style/fonts.scss

```diff
@@ -0,0 +1,14 @@
+@font-face {
+    font-family: "Chivo Light";
+    src: url("../fonts/Chivo-Light.woff2");

```

Comment:
For learning project like this you should use google fonts. It is easier and you don't have download and add font file to repo.

<img width="606" alt="image" src="https://user-images.githubusercontent.com/6817589/155138956-218de7b4-0233-402f-a9dd-1564d9fbed0f.png">

https://fonts.google.com/specimen/Chivo?query=chivo